client/js/jsonedit.js

```diff
@@ -554,6 +554,21 @@ const jeCommands = [
       jeSetAndSelect(w.get('width'));
     }
   },
+  {
+    id: 'je_moveLimit',
+    name: 'move limits',
+    context: '^[^ ]* ↦ moveLimit',
+    call: async function() {
+      const w = widgets.get(jeStateNow.id);
+      jeStateNow.moveLimit = {
+        minX: '###SELECT ME###',
+        maxX: 1600 - w.get('width'),
+        minY: 0,
+        maxY: 1000 - w.get('height'),
+      };
+      jeSetAndSelect(0);
+    }
+  },
```

Comment:
this actually breaks existing moveLimit. I think you should just remove it. It will create more frustration then help

Reply:
I assume you mean that if you already have entered something into the dragLimit property and click on the button in the JSON Editor, it resets what you have with this default.  You are correct it does that and that could be a problem.  But it is useful to have something that fills in the property for you.  I followed the grid example (but of course grid is an array of objects, so adding a new one is no problem).

But there has to be a way to just make the button disappear if dragLimit is already a property.  I just don't know how.

Reply:
This should be like normal JSON option where we just as one property.
remove it or leave as you wish and create an issue so someone can make it properly. I don't really know that part of the JSON editor.